app/javascript/react/components/capes/CapeTile.test.js

```diff
@@ -0,0 +1,69 @@
+import React from "react"
+import Enzyme, { mount } from "enzyme"
+
+import Adapter from "enzyme-adapter-react-16"
+
+Enzyme.configure({ adapter: new Adapter() })
+
+import CapeTile from "./CapeTile"
+import { BrowserRouter } from "react-router-dom"
```

Comment:
I would import all of the "libraries" and config setup at the top, and the import of the cape tile slightly seperate